chord_metadata_service/patients/tests/test_api.py

```diff
@@ -415,34 +437,34 @@ def test_public_filtering_extra_properties_1(self):
     def test_public_filtering_extra_properties_1_config_empty(self):
         # extra_properties string search
         # test GET query string search for extra_properties field
-        response = self.client.get('/api/public?smoking=Non-smoker&death_dc=Deceased')
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        response = self.dt_authz_counts_get('/api/public?smoking=Non-smoker&death_dc=Deceased')
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
         response_obj = response.json()
         self.assertIsInstance(response_obj, dict)
         self.assertEqual(response_obj, dres.NO_PUBLIC_DATA_AVAILABLE)
 
     @override_settings(CONFIG_PUBLIC=DISCOVERY_CONFIG_EXTRA_PROPERTIES)
     def test_public_filtering_extra_properties_2(self):
         # extra_properties string search (multiple values)
-        response = self.client.get(
+        response = self.dt_authz_counts_get(
             '/api/public?smoking=Non-smoker&death_dc=deceased&covidstatus=positive'
         )
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
```

Comment:
why is this 400 now?

Reply:
it was already returning a 400-code error in the JSON body, but with a 200 HTTP response code - this seems confusing, so I made it consistently an error